newm-server/src/main/kotlin/io/projectnewm/server/auth/oauth/OAuthLoginRequest.kt

```diff
@@ -0,0 +1,6 @@
+package io.projectnewm.server.auth.oauth
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class OAuthLoginRequest(val accessToken: String)
```

Comment:
Does the token we give back to the client time bound?

Reply:
This is for the mobile client. The client login using the Google, Facebook or LikedIn SDK. The SDK gives the accessToken that then the client exchanges with the server for our jwt.